### PR TITLE
[CN-1139] Add missing phone home metrics

### PR DESCRIPTION
**Pull request by semihbkgr**

### CODE REVIEW

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 1]
```diff
@@ -55,6 +55,20 @@ The following information is sent in a phone home:
 ** Number of external backups to Google Storage
 ** Number of external backups to Azure Blob Storage
 ** Number of created WAN Sockets using Advanced Network
+* List of allocated resources:
+** Limit of memory
+** Limit of CPU
+* TieredStorage specific:
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/phone-homes.adoc:61]:
```suggestion
* Tiered Storage specific:
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 2]
```diff
@@ -55,6 +55,20 @@ The following information is sent in a phone home:
...
+** Number of maps with tiered storage enabled
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/phone-homes.adoc:62]:
```suggestion
** Number of maps with Tiered Storage enabled
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 3]
```diff
@@ -55,6 +55,20 @@ The following information is sent in a phone home:
...
+* Number of Hazelcast cluster where SQL is configured
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/phone-homes.adoc:63]:
```suggestion
* Number of Hazelcast clusters where SQL is configured
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 4]
```diff
@@ -55,6 +55,20 @@ The following information is sent in a phone home:
...
+* Number of CustomConfig declared
+* Jet specific:
+** Number of Hazelcast cluster where Jet engine is enabled
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/phone-homes.adoc:66]:
```suggestion
** Number of Hazelcast clusters where Jet engine is enabled
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 5]
```diff
@@ -55,6 +55,20 @@ The following information is sent in a phone home:
...
+** Number of Hazelcast cluster where LosslessRestart is enabled
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/phone-homes.adoc:67]:
```suggestion
** Number of Hazelcast clusters where LosslessRestart is enabled
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 6]
```diff
@@ -55,6 +55,20 @@ The following information is sent in a phone home:
...
+* Number of created JetJobSnapshot resources
+* TLS specific:
+** Number of Hazelcast cluster where TLS is enabled
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/phone-homes.adoc:70]:
```suggestion
** Number of Hazelcast clusters where TLS is enabled
```

File: docs/modules/ROOT/pages/phone-homes.adoc [hunk 7]
```diff
@@ -55,6 +55,20 @@ The following information is sent in a phone home:
...
+** Number of Hazelcast cluster where MTLS is enabled
```
Comment (Serdaro) [inline docs/modules/ROOT/pages/phone-homes.adoc:71]:
```suggestion
** Number of Hazelcast clusters where MTLS is enabled
```

**Review by Serdaro — COMMENTED** [review]
Minor suggestions.

**Review by rebekah-lawrence — COMMENTED** [review]
LGTM once Serdar's comments are addressed.

File: docs/modules/ROOT/pages/phone-homes.adoc (hunk 1 above)
Comment (semihbkgr) [reply docs/modules/ROOT/pages/phone-homes.adoc:61]: thanks for the suggestions 👍

**Review by Serdaro — APPROVED** [review]

**Review by rebekah-lawrence — APPROVED** [review]

**Pull request merged**